contributing-guides/style-guide.md

```diff
@@ -160,16 +160,26 @@ All borders of integer and float ranges get included. If you want to exclude the
   using `{{*.ext}}` explains the command without being unnecessarily specific;
   while in `wc -l {{path/to/file}}` using `{{path/to/file}}` (without extension) is sufficient.
 
+### Grouping placeholders
+
+- If a command can take 0 or more arguments of the same kind, use an ellipsis: `{{placeholder1 placeholder2 ...}}`.
+  For instance, if multiple paths are expected `{{path/to/directory1 path/to/directory2 ...}}` can be used.
+- If a command can take 0 or more arguments of different kinds, use an ellipsis: `{{placeholder1|placeholder2|...}}`.
+  If there are more than 5 possible values use `|...` after the last item.
+- To restrict the minimum or (and) maximum placeholder count use range syntax after the ellipsis:
+  For instance if 4 or more paths are expected `{{path/to/directory1 path/to/directory2 ...(4..)}}` can be used.
```

Comment:
I think this syntax will confuse users and they must see this document to learn the meaning of `...(4..)`. Is `...(4..)` a well-known syntax in any programming language? Or can we mention "4 directories are expected" in the description of the example and use `{{path/to/directory1 path/to/directory2 ...}}`? cc @navarroaxel @kbdharun 
```suggestion
  For instance if 4 or more paths are expected `{{path/to/directory1 path/to/directory2 ...(4..)}}` can be used.
```

Reply:
Range syntax is [well known](https://kotlinlang.org/docs/ranges.html) in Kotlin for instance. I don't see syntax with omitted range borders though, but I guess it can be considered as our extension.

Reply:
@blueskyson, @navarroaxel, what do u think about this:

> For instance if 4 or more paths are expected `{{4 or more: path/to/directory1 path/to/directory2 ...}}` can be used.

?